open_dread_rando/templates/custom_init.lua

```diff
@@ -2,31 +2,48 @@ Game.ImportLibrary("system/scripts/init_original.lua")
 
 local initOk, errorMsg = pcall(function()
 
-RemoteLua = RemoteLua or { Init = function() end, }
-RL = RemoteLua
-
-if TEMPLATE("enable_remote_lua") then
-    Game.LogWarn(0, "Starting remote lua")
-    RemoteLua.Init()
+RemoteLua = RemoteLua or { 
+    -- defined by exlaunch
+    Init = function() end,
+    Update = function() end,
+    SendLog = function(message) end,
+    SendInventory = function(message) end,
+    SendIndices = function(message) end,
+    SendReceivedPickups = function(message) end,
+    SendNewGameState = function(message) end,
+    -- defined by rdv
+    GetInventoryAndSend = function() end,
+    GetCollectedIndicesAndSend = function() end,
+    GetReceivedPickupsAndSend = function() end,
+    GetGameStateAndSend = function() end,
+    UpdateRDVClient = function(new_scenario) end
+}
+-- workaround for enable_remote_lua is true but no bootstrap code was received so far
+function RemoteLua.UpdateRDVClient(new_scenario)
```

Comment:
Is there a reason to re-define this?

Reply:
Without the redefine, I always get this error message:
`00:01:09.168 |W| HLE.GuestThread.19 KernelSvc OutputDebugString: Error: [string "system/scripts/scenario.lc"]:257: attempt to call field 'UpdateRDVClient' (a nil value)`

Well, the comment is not accurate cause it also happens with enable_remote_lua set to false.

Reply:
It shouldn't be there in the `RemoteLua = RemoteLua or {...}` then, because that's not something exlaunch defines.

Reply:
That's a good point. Then I will also remove the stubs from what rdv sends as bootstrap code because that is also not defined by exlaunch.